### DISCUSSION TITLE
Config file support

### DISCUSSION THREAD

**Discussion by deekshithkv** [cesanta/mongoose#1861] [Ask questions, get answers] [answered]
Is there config file support to configure (Eg: port, root etc) in mongoose webserver?

**Comment by scaprile** [comment, marked as answer]
Mongoose is a development library, you write code to do what you need
https://mongoose.ws/tutorials/

**Comment by deekshithkv** [comment]
Does latest mongoose version support Digest access authentication?

**Reply by scaprile** [reply]
Not AFAIK https://mongoose.ws/documentation/#mg_http_creds

**Reply by deekshithkv** [reply]
Documents on mongoose states that it support digest authentication? Is it removed in the latest version?
link - https://code.google.com/archive/p/mongoose/
https://cesanta.com/blog/cookie-authentication-and-session-management-mongoose-more-than-an-embedded-web-server/#:~:text=Mongoose%20Embedded%20Web%20Server%20supports,web%20app%20control%20the%20UI.
